.github/workflows/unittest.yml

```diff
@@ -23,7 +23,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: ['3.8', '3.10']
```

Comment:
May be a non-issue but why do we not run unit tests on 3.9?

Is it that we assume our regression tests are exercising all the things the unit tests would anyways, so we don't need to unit test on 3.9?

Is it not possible for some unit test that covers a functionality that is not exercised by regression tests fails on 3.9, which then would not be caught? Or am I overthinking it.

Reply:
We've never actually had any issues with different python versions, and it's probably overkill to test with multiple versions. I'm figuring if there are any issues due to python versions they'll be pretty noticeable, so splitting the versions between unit and regression tests seems fine. Going forward it would probably make more sense to use 1-2 versions of python and possibly test with different versions of jax/numpy/matplotlib etc. But in that case the number of jobs gets pretty huge so I think we're better off only doing that at specific times (like we do now with the jax and mpl version test actions), not on every PR.

Reply:
That's fair, yea I think you're right in it is mainly different jax/mpl versions that cause issues, not different python versions

Reply:
As for this specifically, I just removed it to reduce the number of jobs running (we're only allowed 20 concurrent jobs for the whole PlasmaControl organization), since adding `pytest-split` runs a lot of stuff in parallel. Right now unit tests will take 6 jobs (2 python versions x 3 groups) and regression tests will take 4 jobs (1 python version x 4 groups). We can play around with this more in the future if needed.